components/service/glean/src/test/java/mozilla/components/service/glean/storages/TimespansStorageEngineTest.kt

```diff
@@ -116,109 +96,31 @@ class TimespansStorageEngineTest {
     fun `multiple elapsed times must be correctly accumulated`() {
         val expectedChunkNanos: Long = 37
 
-        val metric = TestMetricData(
+        val metric = TimespanMetricType(
             false,
             "telemetry",
             Lifetime.Ping,
             "single_elapsed_test",
-            listOf("store1"))
-
-        // Initialize a mocked engine so that we can return a custom elapsed time.
-        val spiedEngine = getSpiedEngine()
+            listOf("store1"),
+            timeUnit = TimeUnit.Nanosecond
+        )
 
         // Record the time for the first chunk.
-        doReturn(0.toLong()).`when`(spiedEngine).getElapsedNanos()
-        spiedEngine.start(metric)
-        doReturn(expectedChunkNanos).`when`(spiedEngine).getElapsedNanos()
-        spiedEngine.stopAndSum(metric, TimeUnit.Nanosecond)
+        TimingManager.getElapsedNanos = { 0 }
```

Comment:
Oh, I now see why you reset `TimingManager.getElapsedNanos` in `resetGlean`. Should we instead put this in the `@Before` above? This function should only really be overridden in timing related test files, not all of them.

Reply:
Sure -- I can move it there.  In that case, I think it should be in `@After`, though.